test/test_sql_regexp.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import anymarkup
+from typing import Any, Dict
+from jsonschema.exceptions import ValidationError
+
+from utils import get_schema_object_validator
+
+valid_sql_queries = [
+    "select a from table_a;",
+    "SELECT A FROM TABLE_A;",
+    "select a from table order by id;",
+    "select a from table order by id;",
+    "select a from ( select a,b from table_b );",
+    "explain select a from table_a;"
+]
+
+not_valid_sql_queries = [
+    "select * from table_a",
+    "SELECT * FROM TABLE_A",
+    "select a from ( select * from table_b )",
+    "explain analyze select a from table b"
+]
+
+def new_query_spec(query: str) -> Dict[str, Any]:
+    return {
+        "$schema": "/app-interface/app-interface-sql-query-1.yml",
+        "labels": {},
+        "name": "query",
+        "namespace": {
+            "$ref": "namespace.yaml"
+        },
+        "identifier": "query",
+        "output": "stdout",
+        "query": query,
+    }
+
+@pytest.fixture(scope='session')
```

Comment:
Just to verify that I'm understanding this correctly. The `session` scope actually means that this fixture is only created once in the whole test execution so that `load_schemas` is called just once?

Reply:
Yes. Your comment has made me change the code. I think it's better to have a session scoped fixture that loads the schemas (only happens once in the pytest run) and have another module scoped fixture to get the sql validator. 

I think it makes now and is more reusable for future tests.

Reply:
Yeah, I like it more this way, it is definitely more reusable.